ci/docker/Dockerfile.build.armv7

```diff
@@ -18,7 +18,10 @@
 #
 # Dockerfile to build MXNet for Android ARMv7
 
-FROM mxnetcipinned/dockcross-linux-armv7:11262018
+# The container is pinned for preventing CI failures on updates, swap below to use 
+# the upstream container
+#FROM dockcross/linux-armv7
+FROM mxnetci/dockcross-linux-armv7-pinned
```

Comment:
Should this container be given another tag besides `latest` https://hub.docker.com/r/mxnetci/dockcross-linux-armv7-pinned/tags

Reply:
What do you mean?

Reply:
If you only specify the name `mxnetci/dockcross-linux-armv7-pinned`, instead of `mxnetci/dockcross-linux-armv7-pinned:TAG`, docker will always pull the version tagged as `latest`: `mxnetci/dockcross-linux-armv7-pinned:latest`.
But that version changes whenever someone pushes a new version of `dockcross-linux-armv7-pinned` to `mxnetci`. So it becomes impossible to test if an update of `dockcross-linux-armv7-pinned` breaks the CI. Instead all CI runs will be changed immediately.

If instead you add a tag `TAG` for the particular version of the `mxnetci/dockcross-linux-armv7-pinned` container at https://hub.docker.com/r/mxnetci/dockcross-linux-armv7-pinned/tags, our Dockerfiles here on Github can mention the tag `mxnetci/dockcross-linux-armv7-pinned:TAG`. Then, when pushing a new update of `dockcross-linux-armv7-pinned` to `mxnetci` and tagged as `TAG2`, the CI will continue to use the old version until a PR is opened to update to use `TAG2` on github.

Reply:
I think that's the idea, that we push the container if it can be built, so we don't have to constantly update this file.

Reply:
But then there is no need to use `mxnetci`. Then we can just directly use the `dockcross` repository.
You mentioned that in the past `dockcross` sometimes broke our build. I thought we used pinning to fix that?

Reply:
Mxnetci is a fully automatically handled repository while MXNetcipinned is manually curated. In case there's a security breach, the pinned repo wouldn't be impacted. Since the pinned image usually is no longer available at dockcross since they don't have tagging, the image would then be lost if the image gets deleted by the automatic process. Thus, I'd recommend to stay on the separate pinned repo

Reply:
The pinned image needs to be maintained automatically. It's a good point with the credentials. We might consider having it in a private pipeline with a different repo to preserve those credentials.